src/core/ObservableQuery.ts

```diff
@@ -295,6 +296,7 @@ export class ObservableQuery<
 {
   public readonly options: ObservableQuery.Options<TData, TVariables>;
   public readonly queryName?: string;
+  public variablesUnknown: boolean = false;
```

Comment:
Does this actually need to be public? Can we make this private instead and use `oq['variablesUnknown']` in `QueryManager`?


```suggestion
  private variablesUnknown: boolean = false;
```